main/src/main/assets/full_licenses.html

```diff
@@ -7,7 +7,7 @@ <h2 id="ics-openvpn">OpenVPN for Android</h2>
 <p>
 The name ics-openvpn and OpenVPN for Android refer to the same program.
 OpenVPN for Android is distributed under the GPL license version 2 (see Below)
-with the following clarification/additional terms of what constitues "deriviate work": </p>
+with the following clarification/additional terms of what constitutes "derivative work": </p>
```

Comment:
This appears related to the first one.

For simplicity, I'm going to split all changes to these files into a distinct PR so that they can be separately considered.